pkg/oci/config.go

```diff
@@ -50,8 +50,17 @@ type LoadBalancerConfig struct {
 	// E.g. 10.82.0.0/16 30000-32000
 	DisableSecurityListManagement bool `yaml:"disableSecurityListManagement"`
```

Comment:
This should be deleted right?

Reply:
I've added logging of a depreciation warning and then use it to set `SecurityListManagementMode: ModeManageNone` for backwards compat.